src/Swoole/Actions/EnsureRequestsDontExceedMaxExecutionTime.php

```diff
@@ -3,13 +3,15 @@
 namespace Laravel\Octane\Swoole\Actions;
 
 use Laravel\Octane\Swoole\SwooleExtension;
+use Swoole\Http\Response;
 
 class EnsureRequestsDontExceedMaxExecutionTime
 {
     public function __construct(
         protected SwooleExtension $extension,
         protected $timerTable,
-        protected $maxExecutionTime
+        protected $maxExecutionTime,
+        protected $server
```

Comment:
Even though unlikely, if this action is used directly by users, it'll result in a breaking change.